inc/includes.php

```diff
@@ -40,25 +40,18 @@
 $TIMER_DEBUG = new Timer();
 $TIMER_DEBUG->start();
 
-foreach (['glpi_table_of', 'glpi_foreign_key_field_of'] as $session_array_fields) {
-   if (!isset($_SESSION[$session_array_fields])) {
-      $_SESSION[$session_array_fields] = [];
-   }
-}
 
 /// TODO try to remove them if possible
 include_once (GLPI_ROOT . "/inc/db.function.php");
 
 // Standard includes
 include_once (GLPI_ROOT . "/inc/config.php");
 
+$GLPI_CACHE->removeItems(['table_of', 'foreign_key_field_of']);
```

Comment:
If items are erased on each process, maybe forcing the `Memory` adapter is a better solution.

Reply:
Do not remember but that looks like a dev only line :D

Reply:
No, looks like it was already done before your modifications.

Reply:
`glpi_foreign_key_field_of` is computed using `CommonDBTM::getTable()` function, that can be overrided by items. Maybe it has been made to be sure to do not have cache problem when updating `CommonDBTM::getTable()` results.

Reply:
My bad, it was not cleaning but it was defining an empty array if variables was not set.